eng/pipelines/dotnet-buildtools-prereqs-all.yml

```diff
@@ -7,6 +7,14 @@ trigger:
     - src/*
 pr: none
 
+schedules:
+- cron: "0 5 15 * *"
+  displayName: Nightly run
```

Comment:
"Nightly" feels a little odd here since it is only run once a month.  How about something more descriptive like "Monthly rebuild everything run" ... I'm not in love with that particular wording it is more descriptive.

Reply:
That's what I get for using copy and paste.